bootstrap.sh

```diff
@@ -1 +1,10 @@
 #!/bin/bash
+
+kind create cluster --config kind
```

Comment:
Ensure that the `kind create cluster --config kind` command uses the correct path to the configuration file. It seems like 'kind' should be a file path, e.g., `--config kind-config.yaml`.